modules/claims_api/app/controllers/concerns/claims_api/target_veteran.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'rest-client'
+
+module ClaimsApi
+  module TargetVeteran
+    extend ActiveSupport::Concern
+
+    included do
+      def target_veteran
+        @target_veteran ||= if @is_valid_ccg_flow
+                              build_target_veteran(veteran_id: params[:veteranId], loa: { current: 3, highest: 3 })
```

Comment:
Just a comment for historical tracking, these access tokens are associated with LOA3 accounts only when attended login is completed.